src/App.js

```diff
@@ -1,16 +1,118 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import Todo from './components/TodoComponents/Todo.js'
+
+import './App.css'
+
+const todoListData = [
```

Comment:
interesting, why did you choose to hardcode/write this data here? I see you called it for state so I'm assuming you were just using this as "dummy data' to test/play with or...?